go.mod

```diff
@@ -81,7 +81,7 @@ require (
 	github.com/robfig/cron v1.1.0
 	github.com/spf13/afero v1.2.2
 	github.com/spf13/cobra v1.1.1
-	github.com/spf13/pflag v1.0.5
+	github.com/spf13/pflag v1.0.6-0.20210604193023-d5e0c0615ace
```

Comment:
https://github.com/spf13/pflag/releases 
v1.0.6 is not released yet.
A formal release would be better?

Reply:
there currently isn't a release that includes this commit https://github.com/spf13/pflag/commit/d5e0c0615acee7028e1e2740a11102313be88de1 if we can ping the maintainers of that project to cut a new release then yea using an official version would be cool, either way we're still pinning the version to a specific commit sha which we do for various other dependencies

Reply:
We generally only pin to a sha when versions aren't available. E.g. Lots of dependencies have not transitioned to go modules.